Source/WTF/Scripts/Preferences/WebPreferences.yaml

```diff
@@ -2159,10 +2159,13 @@ SourceBufferChangeTypeEnabled:
   condition: ENABLE(MEDIA_SOURCE)
   defaultValue:
     WebKitLegacy:
+      "USE(GSTREAMER)": false
```

Comment:
This should not be needed, AFAIK WPE/GTK don't support WebKitLegacy.

Reply:
> So playback on Odysse.com works with this patch?

Well, I don't know, but I suspect it should if they check for features properly.